flatpak_builder_lint/checks/flathub_json.py

```diff
@@ -43,3 +45,25 @@ def check(self, manifest: dict) -> None:
                 if modules := manifest.get("modules"):
                     if self._check_if_extra_data(modules):
                         self.errors.append("flathub-json-modified-publish-delay")
+
+        if only_arches := flathub_json.get("only-arches"):
+            if "arm" in only_arches:
+                self.errors.append("flathub-json-deprecated-arm-arch-included")
+            if "i386" in only_arches:
+                self.errors.append("flathub-json-deprecated-i386-arch-included")
```

Comment:
These are not errors in practice.

Reply:
but the goal here is to minimize the presence of `flathub.json`.

Reply:
Errors from this linter will eventually prevent new PRs from being merged. Complaining about deprecated arch used in flathub.json that has no effect on anything related to the build will only contribute to maintainers' frustration.